MekHQ/data/scenariomodifiers/HouseOfficerAir.xml

```diff
@@ -15,77 +14,21 @@
         <deployOffboard>false</deployOffboard>
         <deploymentZones />
         <destinationZone>5</destinationZone>
-        <fixedUnitCount>1</fixedUnitCount>
+        <fixedUnitCount>0</fixedUnitCount>
         <forceAlignment>1</forceAlignment>
         <forceMultiplier>1.0</forceMultiplier>
-        <forceName>House Officer</forceName>
-        <generationMethod>3</generationMethod>
-        <generationOrder>3</generationOrder>
+        <forceName>Joint Force</forceName>
+        <generationMethod>2</generationMethod>
+        <generationOrder>1</generationOrder>
         <maxWeightClass>4</maxWeightClass>
-        <retreatThreshold>50</retreatThreshold>
+        <minWeightClass>2</minWeightClass>
+        <retreatThreshold>0</retreatThreshold>
         <startingAltitude>5</startingAltitude>
+        <objectiveLinkedForces>
+            <objectiveLinkedForce>Player</objectiveLinkedForce>
+        </objectiveLinkedForces>
         <syncDeploymentType>SameEdge</syncDeploymentType>
         <syncedForceName>Player</syncedForceName>
         <useArtillery>false</useArtillery>
     </forceDefinition>
```

Comment:
We're removing the various House Officer, Liaison, and Integrated Allies objectives and instead just having them hook into whatever the players' force objective is.